modules/claims_api/spec/requests/v2/veterans/power_of_attorney_org_request_spec.rb

```diff
@@ -24,17 +24,13 @@
     end
 
     describe 'submit2122' do
-      b64_image = File.read('modules/claims_api/spec/fixtures/signature_b64.txt')
+      File.read('modules/claims_api/spec/fixtures/signature_b64.txt')
```

Comment:
Do we still need this line if the variable to which it was assigned is gone?

Reply:
no we do not, good catch fixing it now